package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "quickblox",
   "description": "QuickBlox JavaScript SDK",
-  "version": "2.5.1",
+  "version": "2.5.2",
```

Comment:
This version of SDK contains one big feature: auto session management, so why don't set 2.6 version?